package.json

```diff
@@ -3,9 +3,10 @@
   "version": "1.0.2",
   "description": "Sophisticated and functionally-minded async with advanced features: coroutines, promises, ES2015 iterables, fantasy-land",
   "main": "dist/creed.js",
-  "jsnext:main": "src/main.js",
+  "jsnext:main": "dist/creed.es.js",
```

Comment:
Ah, if it's on the wiki then it must be fine :-)
I had assumed that tree-shaking could be more efficient if applied to more but smaller modules (compare rollup/rollup#551) but I guess it doesn't make a huge difference here.